model/metrics/dotnet-metrics.yaml

```
@@ -0,0 +1,276 @@
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling [`ThreadPool.ThreadCount`](https://learn.microsoft.com/dotnet/api/system.threading.threadpool.threadcount).
+    instrument: updowncounter
+    unit: "{thread}"
+    stability: experimental
+
+  - id: metric.dotnet.thread_pool.work_item.count
+    type: metric
+    metric_name: dotnet.thread_pool.work_item.count
+    brief: >
+      The number of work items that the thread pool has completed since
+      the process has started.
+    note: >
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling
+      [`ThreadPool.CompletedWorkItemCount`](https://learn.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount).
+    instrument: counter
+    unit: "{work_item}"
+    stability: experimental
+
+  - id: metric.dotnet.thread_pool.queue.length
+    type: metric
+    metric_name: dotnet.thread_pool.queue.length
+    brief: >
+      The number of work items that are currently queued to be processed by the
+      thread pool.
+    note: >
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling
+      [`ThreadPool.PendingWorkItemCount`](https://learn.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount).
+    instrument: updowncounter
+    unit: "{work_item}"
+    stability: experimental
+
+  - id: metric.dotnet.timer.count
+    type: metric
+    metric_name: dotnet.timer.count
+    brief: "The number of timer instances that are currently active."
+    note: >
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling [`Timer.ActiveCount`](https://learn.microsoft.com/dotnet/api/system.threading.timer.activecount).
+    instrument: updowncounter
+    unit: "{timer}"
+    stability: experimental
+
+  - id: metric.dotnet.assembly.count
+    type: metric
+    metric_name: dotnet.assembly.count
+    brief: "The number of .NET assemblies that are currently loaded."
+    note: >
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling
+      [`AppDomain.CurrentDomain.GetAssemblies().Length`](https://learn.microsoft.com/dotnet/api/system.appdomain.getassemblies).
+    instrument: updowncounter
+    unit: "{assembly}"
+    stability: experimental
+
+  - id: metric.dotnet.exceptions
+    type: metric
+    metric_name: dotnet.exceptions
+    brief: "The number of exceptions that have been thrown in managed code."
+    note: >
+      Meter name: `System.Runtime`; Added in: .NET 9.0.
+
+      This metric reports the same values as calling
+      [`AppDomain.CurrentDomain.FirstChanceException`](https://learn.microsoft.com/dotnet/api/system.appdomain.firstchanceexception).
+    instrument: counter
+    unit: "{exception}"
+    stability: experimental
+    attributes:
+      - ref: error.type
```

nit:

the generic `error.type` note and examples are not so useful here

```suggestion
      - ref: error.type
        note: ""
        examples: ['System.OperationCanceledException', 'Contoso.MyException']
```